ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -241,6 +255,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: aws-ipi-private-p3-f28-long-duration-cloud
+  cron: 28 0 27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;NonPreRelease&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
```

Comment:
Don't need `TEST_FILTERS`, it's covered in `chain: openshift-e2e-test-qe-long-duration`

Reply:
done